lua/gatekeeper/staticlib.lua

```diff
@@ -344,13 +344,32 @@ c = ffi.C
 
 local ifaces = require("if_map")
 
+function check_ifaces(front_ports, back_ports)
```

Comment:
Since `check_ifaces()` is called before `init_iface()`, let's copy the test that `ifaces[v1]` and `ifaces[v2]` exist to `check_ifaces()` as well.